src/pages/Dashboard/Portfolio.tsx

```diff
@@ -44,7 +44,6 @@ export const Portfolio = () => {
   const loading = useSelector(selectPortfolioLoading)
   const isLoaded = !loading
 
-  const isRainbowChartsEnabled = useFeatureFlag('RainbowCharts')
```

Comment:
`totalBalancePlusLpHoldings` above needs to go, too.

Reply:
done, thx!